### ISSUE TITLE
Export Large Values Error (exports as large negative integer)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lixun910/geoda#159]
```
I just noticed that there is still a problem with exporting large values.
I have a column with a tract ID as a string and it exports as a very
large negative integer. This happened when exporting selected and
exporting point files.
```

Original issue reported on code.google.com by `lixun...@gmail.com` on 29 Jun 2015 at 11:34

**Comment by jkoschinsky** [comment]
The export of converted variable types (string to integer) is not correct yet: 

1. I converted the GeoID variable from string to integer (results were correct: same numbers except for leading zero)
2. saved the results (results remained correct)
3. Then exported the file and reopened the table: Now the values are no longer the same - they're wrong. Tried this twice to confirm (see mismatch between geoid and st_int and geo_int, which are identical but both wrong)

Will email you the shapefile fo replicate this.

![screen shot 2015-11-10 at 5 13 38 pm](https://cloud.githubusercontent.com/assets/13679743/11077514/8ee0abe8-87ce-11e5-8d60-8a19e5993875.png)

**Comment by lixun910** [comment]
fixed also placed in Gdal/GeoDaMerge18: a workaround fix that make all integer has more than 9 digits a 64-bit Integer.

**Comment by jkoschinsky** [comment]
GeoDa 1.7.87

Ran into this weird issue in OSX Yosemite, 10.10.5:

1. In non-public data I'll email you (in unlocked folder, which I tested by saving the added variable test2), when I convert the Geo_Fips field from string to integer and save or export it, it reverts back to string.

2. Is saves and remains saved after export in Win 7 Professional (64-bit)

<img width="657" alt="screen shot 2016-03-03 at 7 02 06 pm" src="https://cloud.githubusercontent.com/assets/13679743/13515928/b1009172-e172-11e5-9c17-b606b85328db.png">
